baseline.yaml

```diff
@@ -125,6 +125,38 @@ criteria:
     control_mappings: # TODO
     scorecard_probe:
       - blocksDeleteOnBranches
+  - id: OSPS-TEST
+    maturity_level: 1
```

Comment:
I'm going back and forth with myself on the maturity level. Yes, this is unquestionably a Good Idea™ for all projects, but it's also not a practice that's nearly as widely-adopted as we'd like. Is this a criterion that would cause small projects to ignore the rest of the baseline reflexively? I don't know.

Reply:
I think this is a place where we need to decide to what degree we want to lean on privately-run services like GitHub vs supporting a broad audience.  If we lean on GitHub's services, it's possible for most projects to have at least some pre-merge or post-merge CI.  If not, it becomes hard to measure this.

Interestingly, [OSPS-QA-04](https://baseline.openssf.org/#osps-qa-04) at maturity level 2 requires "any automated status checks for commits" to pass or be manually acknowledged, but there is no requirement that "the project MUST have at least one automated status check applied prior to merging changes".

Reply:
As highlighted in PR #131, maturity level 1 should work for code or non-code projects. So I think this one has to be at least level 2?